.github/workflows/ci.yml

```diff
@@ -40,6 +40,10 @@ jobs:
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $RETRY ./mvnw install -B -V -T C1 -DskipTests -P ci -pl '!:trino-server-rpm'
+      - name: Test Server Dependencies
+        run: |
+          export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
+          ./mvnw provisio:validateDependencies ${MAVEN_FAST_INSTALL} -pl :trino-server
```

Comment:
Add a code level comment what it does and why this is important.

(i.e. what to do when this fails and what should i consider if i wanted to change/remove these lines?)

Reply:
Added a comment. The error message would list missing dependencies, like so:
```
Missing dependencies: io.airlift:launcher:tar.gz:properties:205
```
Is this actionable enough?